composer.json

```diff
@@ -56,7 +56,8 @@
 		"npm-asset/fullcalendar": "^3.0.1",
 		"npm-asset/cropperjs": "1.2.2",
 		"npm-asset/imagesloaded": "4.1.4",
-		"pear/console_table": "^1.3"
+		"pear/console_table": "^1.3",
+		"bower-asset/dompurify": "^1.0"
```

Comment:
Can you please move the package name alphabetically along with the other bower packages?